client/src/components/DeleteUserModal/index.tsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { Button, Modal } from 'semantic-ui-react';
+import { useTranslation } from 'react-i18next';
+
+interface Props {
+	onClose: () => void;
+	deleteUser: () => void;
+}
+
+const DeleteUserModal: React.FC<Props> = (props) => {
+	const { deleteUser, onClose } = props;
+	const { t } = useTranslation();
+
+	const submitForm = () => {
+		deleteUser();
+		onClose();
+	};
+
+	return (
+		<Modal closeIcon onClose={() => onClose()} open={true} size={'tiny'} dimmer="inverted">
```

Comment:
if you call func without arguments, you can type it without excess adition arrow func

onClose={onClose}